elastic/kibana
# [Security Solution] Adds tour for new upgrade flyout diff features

## Summary
Issue: https://github.com/elastic/kibana/issues/166489

Adds a tour and tooltips highlighting and describing the new diff features of the prebuilt rule update flyout.

#### To test: 
Enable the `jsonPrebuiltRulesDiffingEnabled` and/or `perFieldPrebuiltRulesDiffingEnabled` feature flags and clear your browser of the local storage `securitySolution.rulesManagementPage.newFeaturesTour.v8.13` token. This should allow you to see the tour and both new tabs highlighted by tooltips as shown in the screenshots below.

### Screenshots
**Upgrade tour**
<img width="1406" alt="Screenshot 2024-02-13 at 2 14 07 PM" src="https://github.com/elastic/kibana/assets/56367316/20d3fd8e-fc39-4ae2-a627-272ae14e9aac">

**Diff tab tooltips**
<img width="1176" alt="Screenshot 2024-02-13 at 2 14 22 PM" src="https://github.com/elastic/kibana/assets/56367316/8810c17e-4dfb-4758-b4c7-199a44531a4e">



### Checklist

Delete any items that are not applicable to this PR.




### For maintainers
